src/api/obsidian-projects-api.ts

```diff
@@ -129,19 +131,21 @@ class ProjectAPI extends ProjectView {
         const localSettings: LocalSettings = {
             ...this.plugin.settings.local_settings,
         }
-        if (project.dataview) {
-            localSettings.source_destination_path = project.path;
+
+        if (project.dataSource.kind === "dataview") {
+            localSettings.source_destination_path = "";
```

Comment:
Leaving this as empty string for now. If it's a dataview project, the `path` doesn't exist.